lib/logstash/inputs/elasticsearch.rb

```diff
@@ -126,7 +133,7 @@ class LogStash::Inputs::Elasticsearch < LogStash::Inputs::Base
   config :docinfo, :validate => :boolean, :default => false
 
   # Where to move the Elasticsearch document information. By default we use the @metadata field.
-  config :docinfo_target, :validate=> :string, :default => LogStash::Event::METADATA
+  config :docinfo_target, :validate=> :field_reference, :default => LogStash::Event::METADATA
```

Comment:
there's an opportunity to change the default here in ECS mode to `[@metadata][input][elasticsearch]` 
so we would end up with: `[@metadata][input][elasticsearch][_index]` etc instead of just `[@metadata][_index]`

Reply:
I like it for consistency with the other plugin metadata. 👍🏼